### DISCUSSION TITLE
help!

### DISCUSSION THREAD

**Discussion by iChezz** [twbs/bootstrap#35208] [Q&A]
![Screenshot_2021_1016_105128](https://user-images.githubusercontent.com/88133027/137573023-46120f92-4c19-418c-8746-2c0aa241d724.jpg)
how to move the "Ninja Warrior" line to the side of "Among Us"? I use bootstrap 5.1

**Comment by mozi-h** [comment]
It looks like you're using cards. You have two out-of-the-box options:

- Card groups for attached cards
- Grid cards for separated cards

Check them out in the [Docs](https://getbootstrap.com/docs/5.1/components/card/#card-layout)

**Comment by adityadees** [comment]
you can use example grid from here [Album Example](https://getbootstrap.com/docs/5.1/examples/album/)

**Comment by carlospetrucio** [comment]
Result:
![image](https://user-images.githubusercontent.com/50377984/138332553-afa2aa6f-1da4-42ac-89b2-e615e2e803ba.png)

### Code for resolution 👍 
`      <div class="container">
        <div class="row">

          <div class="col-sm-6">
            <div class="card">
              <div class="card-body">
                <img src="..." class="card-img-top" alt="...">
                <h5 class="card-img">Title</h5>
                <p class="card-text">All text of card</p>
                <a href="#" class="btn btn-primary">Download</a>
              </div>
            </div>
          </div>

          <div class="col-sm-6">
            <div class="card">
              <div class="card-body">
                <img src="..." class="card-img-top" alt="...">
                <h5 class="card-title">Title</h5>
                <p class="card-text">All text of card</p>
                <a href="#" class="btn btn-primary">Download</a>
              </div>
            </div>
          </div>
        </div>
          </div>

</div>`
